CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to Affinidi CLI
+
+## Getting started
+
+Clone the repository and run `npm install` command.
+
+Some useful commands:
+
+- Use `npm run dev` to run the CLI locally.
+  > To execute commands, prefix them with `npm run dev` (and ommit the `affinidi` name).
+  > Check out the in-depth instructions in the [CLI ReadMe](README.md) file for using existing commands.
+- Run `npm test` to execute unit tests
+  > You can find them in the `src/test/unit-tests` directory.
+- Run `npm run test:integration` to execute integration tests
+  > You can find them in the `src/test/integration-test` directory.
+- `npm run lint` can be used to check for linting errors
```

Comment:
Hi
Thanks for the doc update!
I've tried to run the `test:integration`earlier
but I believe I'm missing some env vars right ?
Can we update the readme to make explain what is missing and how to get/add those infos?
Tks

Reply:
Yes, you need to add the API key and name space for the test email.

Reply:
MAIL_API_KEY=<mail_api_key>
MAIL_NAMESPACE=<mail_namespace>

Reply:
I think the external builder may run integration tests locally, but they will need to get their own MAIL_API_KEY

Reply:
And they'll need to sign up first using this email then run the tests

Reply:
we will add more about tests in a future PR